src/main/java/ch/njol/skript/effects/EffDetonate.java

```diff
@@ -0,0 +1,56 @@
+package ch.njol.skript.effects;
+
+import org.bukkit.entity.*;
```

Comment:
We're not allowed to use star imports

Reply:
IntelliJ did that, I don't know why

Reply:
Apparently we are now. I thought the exact same thing but Sovde said that they were all good now

Reply:
You can disable them in the IntelliJ settings.